### PR TITLE
Adjust specs for shfmt and shellcheck

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: packer-assets/system-info.d/basic.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ commands:
     - { command: clang --version, name: clang version }
     - { command: jq --version, name: jq version }
     - { command: bats --version, name: bats version }
-    - { command: shellcheck --version, name: shellcheck version, pipe: "awk '/^version:/ { print $2 }'" }
+    - { command: shellcheck --version, pipe: "awk '/^version: { print $2 }'", name: shellcheck version }
```
Comment (meatballhat) [inline packer-assets/system-info.d/basic.yml]: :man_facepalming: how'd I manage to break this?

Comment (Lyoness) [inline packer-assets/system-info.d/basic.yml]: `/^version:`  needs a trailing backslash? 

`/^version:/`

Comment (Lyoness) [inline packer-assets/system-info.d/basic.yml]: I should have caught that in review.

**Comment by meatballhat** [comment]
Is happy 👍 https://travis-ci.org/travis-infrastructure/packer-build/jobs/193431891

**Pull request merged**